yt_dlp/extractor/veoh.py

```diff
@@ -123,3 +130,91 @@ def _real_extract(self, url):
             'categories': categories,
             'tags': tags.split(', ') if tags else None,
         }
+
+
+class VeohUserIE(VeohIE):
+    _VALID_URL = r'https?://(?:www\.)?veoh\.com/users/(?P<user>([\da-zA-Z_\-]+))'
+    IE_NAME = 'veoh:user'
+
+    _TESTS = [
+        {
+            'url': 'https://www.veoh.com/users/valentinazoe',
+            'info_dict': {
+                'id': 'valentinazoe',
+                'title': 'valentinazoe (Uploads)'
+            },
+            'playlist_mincount': 75
+        },
+        {
+            'url': 'https://www.veoh.com/users/PiensaLibre',
+            'info_dict': {
+                'id': 'PiensaLibre',
+                'title': 'PiensaLibre (Uploads)'
+            },
+            'playlist_mincount': 2
+        }]
+
+    _USERSINFO_ENDPOINT = 'https://www.veoh.com/users/published/videos'
+    _VIDEO_BASEURL = 'https://www.veoh.com/watch/'
+
+    def _get_authtoken(self, page, uploader=None):
+        webpage = self._download_webpage(page, uploader)
+        return self._search_regex(
+            r'csrfToken: "(?P<token>[0-9a-zA-Z]{40})"', webpage,
+            'request token', group='token')
+
+    def _get_videos(self, uploader, authtoken=None):
+        if authtoken is None:
+            authtoken = self._get_authtoken(f'https://www.veoh.com/users/{uploader}', uploader)
+        totalVids = 0
+        for i in itertools.count():
+            payload = json.dumps({'username': uploader, 'maxResults': 16, 'page': i + 1, 'requestName': 'userPage'}).encode('utf-8')
+            headers = {
+                'x-csrf-token': authtoken,
+                'content-type': 'application/json;charset=UTF-8'
+            }
+            for retry in self.RetryManager():
+                try:
+                    response = self._download_json(
+                        self._USERSINFO_ENDPOINT, uploader, data=payload, headers=headers,
+                        note=f'Downloading videos page {i + 1}')
+                    if not response['success']:
+                        raise ExtractorError('unsuccessful veoh user videos request')
+                    break
+                except ExtractorError as e:
+                    raise e
```

Comment:
this outer try except does nothing?

also I don't think you are making use of the retry manager here (is it necessary?)